src/test/java/RMSysTest.java

```diff
@@ -0,0 +1,96 @@
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.*;
+
+public class RMSysTest extends TestBase {
+
+    @Test
+    public void fillingLoginForm() {
+        open("https://192.168.100.26/");
+        $("#Username").setValue("EugenBorisik");
+        $("#Password").setValue("qwerty12345");
+        $("#SubmitButton").click();
+        SoftAssert softAssert = new SoftAssert();
+        softAssert.assertNotNull(".sign-out-span>a");
```

Comment:
This code doesn't check anything. Please ask me, if it's not clear why this code is senseless
By the way, replace this code with Selenide should() method